### PR TITLE
[Antora] [PGSQL] Architecture section for postgres doc

### PR DESCRIPTION
**Pull request by vttranlina**

### CODE REVIEW

**Comment by chibenwa** [comment]
If you commit a png also commit the sources that you used for generating it.

**Comment by chibenwa** [comment]
MinIO => S3 please it is more generic

**Review by chibenwa — COMMENTED** [review]
Did not review fully but happy to see dark magic going on with partials

**Review by Arsnael — COMMENTED** [review]
Overall looks good :)

**Comment by vttranlina** [comment]
> If you commit a png also commit the sources that you used for generating it.

yes, I did, 
storage_james_postgres.png -> storage_james_postgres.svg

File: docs/modules/servers/partials/architecture/consistency-model.adoc [hunk 1]
```diff
@@ -18,10 +18,10 @@ This section details this trade-off for data stores used by the {server-name}.
 
 === OpenSearch consistency model
 
-OpenSearch relies on link:https://www.elastic.co/blog/a-new-era-for-cluster-coordination-in-elasticsearch[strong consistency]
-with home grown algorithm.
+OpenSearch relies on link:https://opensearch.org/docs/latest/tuning-your-cluster/[strong consistency]
+with home-grown algorithm.
 
-The 6.x release line, that the distributed server is using is known to be slow to recover from failures.
+The 1.x release line, that the distributed server is using, is known to provide faster recovery.
```
Comment (Arsnael) [inline docs/modules/servers/partials/architecture/consistency-model.adoc:24]: No we using the version 2.x of opensearch :)

**Review by quantranhong1999 — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

File: docs/modules/servers/pages/distributed/architecture/consistency_model_data_replication_extend.adoc [hunk 2]
```diff
@@ -0,0 +1,41 @@
+=== Cassandra consistency model
```
Comment (hungphan227) [inline docs/modules/servers/pages/distributed/architecture/consistency_model_data_replication_extend.adoc:1]: The title here makes me confused. We have an another file called consistency-model.adoc. Why not putting these stuff in that file?

Comment (Arsnael) [reply docs/modules/servers/pages/distributed/architecture/consistency_model_data_replication_extend.adoc:1]: I think what @vttranlina does here is that it allows him to inject what he wants. This is for cassandra in particular, I think there is a similar file for postgres.

Putting that in the consistency-model.adoc directly would be potentially hard to manage between Cassandra and Postgres, as those subsection would be quite different with both implementation.

@vttranlina am I correct?

**Comment by Arsnael** [comment]
Squash fixups please :)

File: docs/modules/servers/pages/distributed/architecture/consistency_model_data_replication_extend.adoc (hunk 2 above)
Comment (vttranlina) [reply docs/modules/servers/pages/distributed/architecture/consistency_model_data_replication_extend.adoc:1]: yes, absolute correct

**Pull request merged**